Casks/font-sarasa-gothic.rb

```diff
@@ -1,8 +1,8 @@
 cask "font-sarasa-gothic" do
   version "1.0.6"
-  sha256 "ce842295201de41a4b09b4f153405aba975cd5553110dbcd11cf69845545528a"
+  sha256 "3246e6127e2f5765840c2f295be59ec0fb958652cf8b68869a1bc95ba4b392cc"
 
-  url "https://github.com/be5invis/Sarasa-Gothic/releases/download/v#{version}/Sarasa-TTC-#{version}.7z"
+  url "https://github.com/be5invis/Sarasa-Gothic/releases/download/v#{version}b/Sarasa-TTC-#{version}.zip"
```

Comment:
Possibly worth using the SuperTTC version, since it's available?

```suggestion
  url "https://github.com/be5invis/Sarasa-Gothic/releases/download/v#{version}b/Sarasa-SuperTTC-#{version}.zip"
```